flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/codegen/agg/batch/HashAggCodeGenerator.scala

```diff
@@ -202,11 +299,43 @@ class HashAggCodeGenerator(
          |    $dealWithAggHashMapOOM
          |  }
          |}
+         |
+         |$totalCountIncCode
+         |$adaptiveSamplePointCode
+         |
          | // aggregate buffer fields access
          |${ctx.reuseInputUnboxingCode(currentAggBufferTerm)}
          | // do aggregate and update agg buffer
          |${aggregate.code}
+         | // flush result form map if suppress is enable. 
+         |$flushResultIfSuppressEnableCode
          |""".stripMargin.trim
+    } else {
+      s"""
```

Comment:
We don't need this repeated code, for local and global agg, the main code is reusable. For example, the `adaptiveSuppressCode ` for global agg is "", but for local agg is the needed code.